pwiz_tools/Skyline/TestRunnerLib/RunTests.cs

```diff
@@ -501,11 +505,50 @@ public static HeapAllocationSizes[] GetProcessHeapSizes()
                 {
                     var h = buffer[i];
                     HeapLock(h);
+                    sizes[i].CommittedSizes = new Dictionary<long, int>();
```

Comment:
I think you are going to need a flag to keep from doing this every time we calculate heap sizes. I think it will be too slow to always collect all of the strings. We had to turn off just to total committed by counts for the perf tests, because that was too slow. Please add a flag that can be turned on in the debugger to have this analysis, but otherwise skip it. i.e. it only happens during an active diagnosis.

Reply:
I wrapped the code into a conditional block with a flag but the debugger is giving me issues when I do this (hangs on "evaluating local variables", then gives an error saying it timed out). Trying to figure out how to work around it.